### ISSUE TITLE
[Bug] Preferences: Not-yet-existing configuration COB_INCLUDE_PATH

### ISSUE THREAD

**Issue by GitMensch** [OpenCobolIDE/OpenCobolIDE#221]
I'm not sure where you got this entry from, but it doesn't exist (neither in OC 1.1 nor in GC).

The only working options to add/change the include path passed to the C compiler are:
* via environment ``set COB_CFLAGS=/Iyourpath``
* via command line option to cobc ``-I yourpath``

But I've added this to the wish-list tracker for GC in [#88][1], as an option COB_INCLUDE_PATH is reasonable (it simply won't apply to any existing [not manually changed] installation).
The new setting will likely be included in the next GC2 "nightly" (expected until 2015-10-25).

[1]:https://sourceforge.net/p/open-cobol/feature-requests/88/

## System information

- OpenCobolIDE: 4.6.6
- pyqode.core: 2.6.9
- pyqode.cobol: 2.6.4
- pyqode.qt: 2.7.dev0
- pygments: 2.0.2

**Comment by ColinDuquesnoy** [comment]
Thanks for the clarification. 

But I am not sure what to do about this "bug". Since the option will be available soon, isn't it better to leave it as it is?

**Comment by GitMensch** [comment]
Maybe add a warning...
Best would be to call ``cobc --info`` and see if this option is available or not (older versions don't have ``cobc --info`` at all, current versions don't have an entry for ``COB_INCLUDE_PATH``).

**Comment by ColinDuquesnoy** [comment]
> Best would be to call cobc --info and see if this option is available or not 

Sounds good. I am going to implement this after #215

**Comment by ColinDuquesnoy** [comment]
I won't fix this in OCIDE.
